rest-service/manager_rest/utils.py

```diff
@@ -275,3 +278,32 @@ def send_event(event, message_type):
 
     events_publisher.publish_message(event, message_type)
     events_publisher.close()
+
+
+def traces(func_name=None):
+    """Wrapper function for Flask operation call tracing.
+    This decorator must be activate iff the following condition apply. One of
+    the parent calls must be done inside a 'with span_in_context(span)' scope
+    (otherwise you'd get a parentless span).
+
+    :param func_name: name to display in tracing
+    """
+
+    def decorator(f):
+        name = func_name
+        if func_name is None:
+            name = f.__name__
+
+        @wraps(f)
+        def with_tracing_wrapper(*args, **kwargs):
+            tracer = getattr(current_app, 'tracer', None)
+            if not tracer:
+                return f(*args, **kwargs)
+            root_span = get_current_span()
+            with tracer.start_span(name, child_of=root_span) as span:
+                with span_in_context(span):
+                    return f(*args, **kwargs)
+
+        return with_tracing_wrapper
+
```

Comment:
in here too

Reply:
That's a PEP8 thing, newline at the end of files.